src/bin/miri-rustc-tests.rs

```diff
@@ -47,7 +47,7 @@ impl rustc_driver::Callbacks for MiriCompilerCalls {
                 impl<'a, 'tcx: 'a, 'hir> itemlikevisit::ItemLikeVisitor<'hir> for Visitor<'a, 'tcx> {
                     fn visit_item(&mut self, i: &'hir hir::Item) {
                         if let hir::ItemKind::Fn(.., body_id) = i.node {
-                            if i.attrs.iter().any(|attr| attr.name() == "test") {
+                            if i.attrs.iter().any(|attr| attr.ident_str() == Some("test")) {
```

Comment:
```suggestion
                            if i.attrs.iter().any(|attr| attr.check_name("test")) {
```

Reply:
that "marks the attribute as used", is that really what we want?

Reply:
I can't think of a way it would hurt.